modules/apps/headless-apio/blog/blog-apio-client-test/src/testIntegration/java/com/liferay/blog/apio/client/test/BlogApioTest.java

```diff
@@ -331,6 +414,24 @@ private String _read(String fileName) throws Exception {
 		return StringUtil.read(url.openStream());
 	}
 
+	private File _readFile(String fileName) {
```

Comment:
I think we have this code is more than 3 places, should we move it to a utility class?